lib/vsc/filesystem/operator.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: latin-1 -*-
+#
+# Copyright 2009-2022 Ghent University
+#
+# This file is part of vsc-filesystems,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-filesystems
+#
+# All rights reserved.
+#
+"""
+Interface to dynamically load vsc.filesystem modules and instantiate its XxxxOperations class
+
+@author: Alex Domingo (Vrije Universiteit Brussel)
+"""
+import importlib
+import logging
+
+STORAGE_OPERATORS = {
+    'posix': ('PosixOperations', 'PosixOperationError'),
+    'gpfs': ('GpfsOperations', 'GpfsOperationError'),
+    'oceanstor': ('OceanStorOperations', 'OceanStorOperationError'),
+    'lustre': ('LustreOperations', 'LustreOperationError'),
+}
+
+
+class StorageOperator:
```

Comment:
use `(Object)` ?

Reply:
do we still support Python 2? :wink: fixed in https://github.com/hpcugent/vsc-filesystems/pull/83/commits/119c37934c50326434580d42c382398a0af9d10b